certbot/main.py

```diff
@@ -363,6 +364,33 @@ def _init_le_client(config, authenticator, installer):
     return client.Client(config, acc, authenticator, installer, acme=acme)
 
 
+def register(config, unused_plugins):
+    """Create or modify accounts on the server."""
+
+    # Currently, only --update-registration is implemented. Issue #2446
+    # calls for a fuller register verb, to allow better separation of
+    # account management from obtaining certs.
+    if not config.update_registration:
```

Comment:
Since we have more time now, we may as well get the register verb to actually register :).  I think it should be very easy to do so; just grab the relevant portion from the beginning of `run` or `obtain_cert`.